src/stores/tests/chat-slice.test.ts

```diff
@@ -228,7 +228,8 @@ describe( 'chat-slice', () => {
 		} );
 
 		it( 'should handle invalid JSON in localStorage gracefully', () => {
-			const consoleErrorSpy = jest.spyOn( console, 'error' );
+			// Silence `console.error` output
+			const consoleErrorSpy = jest.spyOn( console, 'error' ).mockImplementation( () => {} );
```

Comment:
Hm, I think it's better to do it in `beforeEach / afterEach`, to avoid case when this test failed, `consoleErrorSpy.mockRestore();` wasn't run and the next test case fails or works incorrectly due to not restoring this spy. WDYT?

Reply:
That's fair 👍